csrc/scheduler/hopper_multi_matmul.cpp

```diff
@@ -545,12 +545,9 @@ void HopperMultipleMatmulScheduler::run() {
   // This also collects mma_results_
   defineOperandCaches();
 
-  // Schedules:
-  //   - global->smem (cp.async)
-  //   - smem->register (ldmatrix)
-  //   - prologue computation in registers, including broadcast to e.g.
-  //   ab=[iM, bN, iK]
-  schedulePrologues();
+  inspectPrologues();
+
+  scheduleOperandSmemStores();
```

Comment:
For symmetry with `scheduleOperandToRegisters()`.
```suggestion
  scheduleOperandToSmem();
```

Reply:
We currently have `cacheOperandsToSmem` and once #3380 is merged, `cacheOperandsToRegisters`. These are creating `LoadStoreOp`s to do the copies and the `To` refers to the output of that expression. In the case of the `scheduleOperandSmemStores` we are operating on the `TensorView` instead of its definition, so maybe just `scheduleOperandsSmem` or even `scheduleOperands`? In this scheduler there is only one operand tensor to schedule and that lies in smem so it's unambiguous.

Reply:
I like `scheduleOperands`.